include/crow/mime_types.h

```diff
@@ -0,0 +1,116 @@
+//This file is generated from nginx/conf/mime.types using nginx_mime2cpp.py
```

Comment:
well I put it into a separate file on the assumption that using amalgamate is an option, and since this file is required for the automatic Content-Type header setting, I added the file to /include/crow